PENDING.md

```diff
@@ -70,3 +70,6 @@ BUG FIXES
 
 * Tendermint
   * [\#2797](https://github.com/tendermint/tendermint/pull/2797) AddressBook requires addresses to have IDs; Do not crap out immediately after sending pex addrs in seed mode
+
+* Codebase
+  * [\#2798](https://github.com/cosmos/cosmos-sdk/issues/2798) Governance API has miss-spelled English word in JSON response 
```

Comment:
This should go in the "SDK" section. Also could you point out that its specifically `depositer -> depositor`? Makes it easier for people following the changelog :)

Reply:
Hmmmm, tbh I'm not convinced every single little detail should be included in the changelog. Otherwise, the log would get extremely noisy and verbose. Just basing this off of other major OSS projects.

Reply:
^ tots agree, i don't think fixing spelling mistakes is changelog worthy

Reply:
This is a breaking change for JSON responses which users may depend on.

Reply:
ehhh I guess if they're parsing JSON manually, then yeah.

Reply:
Umm, sorry I didn't see this conversation and already just applied the 2 items of feedback. Let me know if you want further changes. Would agree this is changelog worthy as it is a break in the public contract.

Reply:
@alexanderbez no worries! 🙌